docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+# docker-compose.yml
+services:
+
+  app:
+    image: "ghcr.io/remla23-team15/app:latest"
+    ports:
+      - 8080:80
+    depends_on:
+      - model-service
+
+  model-service:
+    image: "ghcr.io/remla23-team15/model-service:latest"
+    ports:
+      - 8081:80
```

Comment:
`model-service` exposes port 8080, so this line should become:

- 8081:8080

Reply:
I see, thanks!